packages/framer-motion/src/context/MotionConfigContext.tsx

```diff
@@ -25,6 +25,14 @@ export interface MotionConfigContext {
      * @public
      */
     transition?: Transition
+
+    /**
+     * If true, will respect the device prefersReducedMotion setting by switching
+     * transform animations off.
+     *
+     * @public
+     */
+    respectReducedMotion?: boolean
```

Comment:
It seems to me that it would be better to have three options here: 
* always reduce motion
* never reduce motion
* listen to prefersReduceMotion
Maybe we can make the API reflect those?


Something like:

```suggestion
    reducedMotion?: "always" | "never" | "userPreference"`
```

Reply:
What's the use-case for `"always"`?

Reply:
Maybe it doesn't make a lot of sense to use an animation framework and disable the animations 😆, but I was thinking about giving the developer control over this feature, like being able to offer a toggle to the user to enable/disable reduced motion.

Reply:
I was mostly trying to future-proof this API, because there is a third option that might be usefull at some point.

Reply:
Ah I see, yeah that's a good idea. How about to keep the cleanliness of 

```jsx
<MotionConfig respectReducedMotion>
```

And to keep the potential Framer settings page toggle in mind, we make this a boolean and then in a later run extend `true` and `false` with

```jsx
<MotionConfig respectReducedMotion="enforce">
```

Reply:
Actually on further thought if we made it `"user"` I think this is a good change, we can still surface it as a boolean in Framer

Reply:
I lost track, can you write out the API you have in mind now?

Reply:
Ah, I see you already implemented it, yes, this looks good!